db/migrate/20200819084304_add_confirmed_number_of_places_to_allocations.rb

```diff
@@ -0,0 +1,5 @@
+class AddConfirmedNumberOfPlacesToAllocations < ActiveRecord::Migration[6.0]
+  def change
+    add_column :allocation, :confirmed_number_of_places, :integer, null: true
```

Comment:
Don't we want to default this to 0? Otherwise we'd have to check for `nil` all the time if using this attribute in calculations.

Reply:
I think we want nil/null here. So we know the the data hasn't been populated. This is a placeholder and will be populated with data once it's been handed to us

Reply:
FYI this was discussed offline